backend/__tests__/models/stays.spec.ts

```diff
@@ -91,7 +91,7 @@ describe('Stay DB functions', () => {
 
   test('findAllStays finds all upcoming guests of a user', async () => {
     // Arrange
-    const extIt = '1'; // Harald Junke
+    const extIt = 1; // Harald Junke
```

Comment:
did extIT change in our db as well? Wasn't it a string?

Reply:
extIt is a string. but the function findAllStays takes a number and actually searches for the user Id.

Reply:
Ah, thought I missed something. Thanks!